xdsl/dialects/onnx.py

```diff
@@ -201,6 +239,7 @@ def verify_(self) -> None:
         # check shape compatibility
         tensor_a_shape = tensor_a_type.get_shape()
         tensor_b_shape = tensor_b_type.get_shape()
+        tensor_c_shape = list(tensor_c_type.get_shape())
```

Comment:
why list?

Reply:
i had this as list, so it match the type of the broadcast function (as its an input for it), but this is now changed